src/filter-object/filter-object.ts

```diff
@@ -0,0 +1,119 @@
+import { IProductFilter } from '../types/product.interface';
+export const filterProperty: IProductFilter = {
```

Comment:
why do you use this hardcoded object here? Could you create it using fetched data?

Reply:
it was done for speed. And it was planned to change.